### ISSUE TITLE
can insert multiple pictures into one cell ?

### ISSUE THREAD

**Issue by JohnnyTing** [qax-os/excelize#803]
```text
 hello, i may need a feature that can insert multiple images into one cell,   have this feature now?
```

**Comment by xuri** [comment]
Yep, please reference the document, you can specify scale, offset, size, and position by [`AddPicture`](https://xuri.me/excelize/zh-hans/image.html)

**Comment by JohnnyTing** [comment]
> Yep, please reference the document, you can specify scale, offset, size, and position by [`AddPicture`](https://xuri.me/excelize/zh-hans/image.html)

**Comment by JohnnyTing** [comment]
@xuri thanks, i got it

**Comment by JohnnyTing** [comment]
@xuri hello, how to retrieve mutiple picture from one cell?  i'm not find  the related api

`func (f *File) GetPicture(sheet, cell string) (string, []byte, error)`, this return first picture []byte?

**Comment by xuri** [comment]
That does not support extracting multi-image by GetPicture currently, I'll consider adding this feature later.

**Comment by JohnnyTing** [comment]
> That does not support extracting multi-image by GetPicture currently, I'll consider adding this feature later.

ok, and have other method to finish it now?

**Comment by xuri** [comment]
This issue has been fixed by PR #1389 and #1501.
